includes/class-edd-cli.php

```diff
@@ -602,7 +602,8 @@ public function payments( $args, $assoc_args ) {
 					'quantity' => 1,
 					'options'  => $options
 				);
-
+				
+				$cart_details = array();
```

Comment:
This is going to reset the `$cart_details` as an array for every item in this `foreach`. It means that we'll never be able to have anything but the last key in this array...were you meaning to set `$cart_details[$key] = array()`? Or maybe put `$cart_details = array();` outside of the foreach?

Foreach starts here:
https://github.com/easydigitaldownloads/Easy-Digital-Downloads/blob/issue/3130/includes/class-edd-cli.php#L574